### ISSUE TITLE
How to deal with large browserified bundles?

### ISSUE THREAD

**Issue by fergiemcdowall** [substack/browserify-handbook#76]
Where do you begin when you want to analyse and fix bundles that are too large? What steps would you carry out to see what is taking up the most room in a broswerified bundle?

**Comment by emilbayes** [comment]
You might have luck trying https://www.npmjs.com/package/disc
Note that it doesn't support ES6 last time I used it

**Comment by fergiemcdowall** [comment]
Disc is cool. Is there a command line tool that does something similar?

**Comment by emilbayes** [comment]
disc is a cli tool, but you mean one that doesn't show a web interface?

**Comment by fergiemcdowall** [comment]
Yes, I would ideally like a tool that can be invoked and parsed on the command line

**Comment by emilbayes** [comment]
It doesn't seem too hard to rip out of disc if you wanted to: https://github.com/hughsk/disc/blob/master/index.js
